odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -319,7 +319,7 @@ Connecting to your virtual machine
 
     $ ssh -i PATH_TO_PRIVATE_KEY USERNAME@IP_ADDRESS
 
-  The first parameter represents the *path to your private key* you used for SSH authentication (in case of automatic generation through azure, it is the path of the key pair .pem file downloaded earlier in your computer), the second parameter *the username* you used for SSH authentication, and the final parameter *the IP address* of the virtual machine.
+  The first parameter represents the *path to your private key* you used for SSH authentication (in case of automatic generation through AZURE, it is the path of the key pair .pem file downloaded earlier in your computer), the second parameter *the username* you used for SSH authentication, and the final parameter *the IP address* of the virtual machine.
```

Comment:
Sorry my last comment was not clear, I think it is "Azure" not "AZURE" or "azure"